### PR TITLE
Fast scrolling

### PR DESCRIPTION
**Pull request by Comum**
Fast scrolling

### CODE REVIEW

File: src/HeavenScroll.js [hunk 1]
```diff
@@ -165,18 +164,35 @@ class HeavenScroll {
...
             pageClassName: this.options.pageClassName,
             pageNumber: printPageNumber
         }
 
+        if (printPageNumber.constructor !== Array) {
+            $page = this.$el.find('[data-page-number=' + printPageNumber + ']');
+            if ($page.hasClass('visibility-hidden') && $page.hasClass('beforePlaceHolderDiv')) {
+                $page
+                    .removeClass('visibility-hidden beforePlaceHolderDiv js-page-hook')
+                    .addClass('js-page-hook');
```
Comment (aaddrriiaann) [inline src/HeavenScroll.js]: Why do you remove then add this class back?

Comment (Comum) [reply src/HeavenScroll.js]: That way I can only target the first page with content being displayed, here: `let pages = document.getElementsByClassName('js-page-hook');`

File: src/HeavenScroll.js [hunk 2]
```diff
@@ -165,18 +164,35 @@ class HeavenScroll {
...
         args = {
             pageClassName: this.options.pageClassName,
             pageNumber: printPageNumber
         }
 
+        if (printPageNumber.constructor !== Array) {
+            $page = this.$el.find('[data-page-number=' + printPageNumber + ']');
+            if ($page.hasClass('visibility-hidden') && $page.hasClass('beforePlaceHolderDiv')) {
+                $page
+                    .removeClass('visibility-hidden beforePlaceHolderDiv js-page-hook')
```
Comment (guillaume-thisplace) [inline src/HeavenScroll.js]: Why remove `js-page-hook` just to add it again?

File: src/HeavenScroll.js [hunk 3]
```diff
@@ -289,24 +305,24 @@ class HeavenScroll {
      * @param {String} position
      */
     removePage(position) {
-        let pageHeight;
-        let html;
-        let $page;
+        let pageHeight; // not used
```
Comment (guillaume-thisplace) [inline src/HeavenScroll.js]: Please remove unused variables.

File: src/HeavenScroll.js [hunk 4]
```diff
@@ -289,24 +305,24 @@ class HeavenScroll {
...
+        let html; // not used
+        let $page; // not used
+        let className = 'visibility-hidden';
 
         if ((position !== 'first') && (position !== 'last')) {
             console.error('removePage(position): "' + position + '" is not a valid argument.');
         } else {
```
Comment (guillaume-thisplace) [inline src/HeavenScroll.js]: Please remove the else block, unindent its contents and `return` after the `console.error()`. This improves readability and prevents bugs where someone adds code after the `else` block that'd be run even if there is an error.

File: src/HeavenScroll.js (hunk 1 above)
Comment (aaddrriiaann) [reply src/HeavenScroll.js]: But on line `179` you remove the class then on line `180` you add the class back. I just don't understand why you do this 😅

File: src/HeavenScroll.js (hunk 2 above)
Comment (Comum) [reply src/HeavenScroll.js]: That way I can target the first page with content being displayed, here: 
```
let pages = document.getElementsByClassName('js-page-hook');
let pageNumber = parseInt(pages[0].getAttribute('data-page-number'));

File: src/HeavenScroll.js (hunk 4 above)
Comment (Comum) [reply src/HeavenScroll.js]: 👍

File: src/HeavenScroll.js (hunk 2 above)
Comment (guillaume-thisplace) [reply src/HeavenScroll.js]: OK just liaise with Adrian on that one

File: src/HeavenScroll.js (hunk 1 above)
Comment (guillaume-thisplace) [reply src/HeavenScroll.js]: @Comum do you do this systematically so that it'll add the `js-page-hook` if it wasn't there in the first place?

Comment (Comum) [reply src/HeavenScroll.js]: My bad, in line `179` `js-page-hook` does not exist. Nice catch.

**Pull request merged**